operators/osdk-cli-reference.adoc

```diff
@@ -0,0 +1,32 @@
+// This assembly is included in the following assemblies:
```

Comment:
After recent discussions, we can probably drop this section at the top of the assembly since we're not nesting them.